### PR TITLE
Restrict filters to images

### LINKED ISSUE #54: Restrict asset selector to show up only images
Since we only support images in phase-1, we need pre-configure asset selector to show only images to avoid confusing end users.

### PR DESCRIPTION
**Pull request by sdmcraft**
Fix #54 

Test URLs:
- Before: https://main--franklin-assets-selector--hlxsites.hlx.live/aem-asset-selector
- After: https://i-56--franklin-assets-selector--hlxsites.hlx.live/aem-asset-selector

### CODE REVIEW

File: blocks/aem-asset-selector/aem-asset-selector-util.js [hunk 1]
```diff
@@ -378,6 +378,56 @@ export async function renderAssetSelectorWithImsFlow(cfg) {
...
+                    {
+                      label: 'PNG',
+                      value: 'image/png',
+                    },
+                    {
+                      label: 'TIFF',
+                      value: 'image/tiff',
+                    },
+                    {
+                      label: 'GIF',
```
Comment (tmathern) [inline blocks/aem-asset-selector/aem-asset-selector-util.js:420]: Do we really want gif? I know users rarely have animated ones, but still. Handling of animated gifs is poor.

File: blocks/aem-asset-selector/aem-asset-selector-util.js [hunk 2]
```diff
@@ -378,6 +378,56 @@ export async function renderAssetSelectorWithImsFlow(cfg) {
...
+                      value: 'image/gif',
```
Comment (tmathern) [inline blocks/aem-asset-selector/aem-asset-selector-util.js:421]: We can also add webp here once we've tested it works with getting high res assets (canvas supports it, we just didn't have the assets to test and I don't have enough permissions to add assets...).

**Review by tmathern — APPROVED** [review]
Approving - but with a caveat: does it let us still see folders? Because without a good tree navigation, showing folders in the same view is the only way we got to navigate.

**Comment by sdmcraft** [comment]
> Approving - but with a caveat: does it let us still see folders? Because without a good tree navigation, showing folders in the same view is the only way we got to navigate.

Good call out! Yes it seems to remove the folders from view. Following up [on this here](https://cq-dev.slack.com/archives/C01RNQM16MP/p1692983026224989?thread_ts=1692862284.982889&cid=C01RNQM16MP).

**Pull request merged**